doc/man3/flux_shell_rpc_pack.adoc

```diff
@@ -0,0 +1,48 @@
+flux_shell_rpc_pack(3)
+======================
+:doctype: manpage
+
+NAME
+----
+flux_sell_rpc_pack - Use Jansson syntax to make a remote procedure call
+
+SYNOPSIS
+--------
+
+ #include <flux/shell.h>
+ #include <errno.h>
+
+ flux_future_t *flux_shell_rpc_pack (flux_shell_t *shell, 
+                                     const char *method, 
+                                     int shell_rank, 
+                                     int flags, 
+                                     const char *fmt, 
+                                     ...);
+
+DESCRIPTION
+-----------
+
+Sends a remote procedure call `method` to rank `shell_rank`.
+
+RETURNS
+-------
+
+Returns NULL and sets `errno` to EINVAL if `shell`, `method` or `fmt` 
+are NULL, or if `rank` is less than 0.  Otherwise, returns the results
+of the underlying `shell_svc_vpack()` call.
```

Comment:
For consistency with other manpages, split into `RETURN VALUE` and `ERRORS`

Reply:
Already done.